standard/ERCs/erc-6900.md

```diff
@@ -144,25 +148,26 @@ Standard execute functions SHOULD check whether the call's target implements the
 
 ```solidity
 struct Call {
-    // The target contract for account to execute.
+    // The target address for the account to call.
     address target;
-    // The value for the call.
+    // The value sent with the call.
```

Comment:
nit: the comments for target and data are in the future tense, but this is past tense. Maybe "to send with" ?